components/director/cmd/main.go

```diff
@@ -166,3 +176,17 @@ func configureLogger() {
 	})
 	log.SetReportCaller(true)
 }
+
+func getTenantMappingHanderFunc(transact persistence.Transactioner, staticUsersSrc string, scopeProvider *scope.Provider) (func(writer http.ResponseWriter, request *http.Request), error) {
+	uidSvc := uid.NewService()
+	authConverter := auth.NewConverter()
+	systemAuthConverter := systemauth.NewConverter(authConverter)
+	systemAuthRepo := systemauth.NewRepository(systemAuthConverter)
+	systemAuthSvc := systemauth.NewService(systemAuthRepo, uidSvc)
+	staticUsersRepo, err := tenantmapping.NewStaticUserRepository(staticUsersSrc)
```

Comment:
BTW I think our approach to load this configmap is confusing. The other configuration we load every X minutes, and this one we load just once, on Director start.

I'm not saying to do it now, but please create follow-up task to also load static users periodically.

Reply:
I will refactor it, generally, I don't like the way our `main.go` looks like. Will try to do something with it.

Reply:
I think we can continue this topic in the issue already defined by https://github.com/kyma-incubator/compass/issues/386